### PR TITLE
Update to work with sqsconsumer v2

### PR DESCRIPTION
**Pull request by jharlap**
Also adds timestamps to log messages

PLAT-5420

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -62,7 +63,7 @@ func main() {
 	var logger log.Logger
 	{
 		logger = log.NewJSONLogger(log.NewSyncWriter(os.Stdout))
-		logger = log.NewContext(logger).With("service", "sqsexec", "version", Version)
+		logger = log.NewContext(logger).With("service", "sqsexec", "version", Version, "timestamp", log.DefaultTimestampUTC)
```
Comment (thebigjc) [inline main.go]: \o/

File: main.go [hunk 2]
```diff
@@ -112,25 +113,29 @@ func main() {
...
+		})
+
+		stack = []middleware.MessageHandlerDecorator{ageTracker, timing}
 	}
 
 	h := middleware.ApplyDecoratorsToHandler(w.HandleMessage, stack...)
 	c := sqsconsumer.NewConsumer(sqsSvc, h)
+	c.ExtendVisibilityTimeoutBySeconds = 600
+	c.ExtendVisibilityTimeoutEvery = 7 * time.Minute
+	c.ReceiveVisibilityTimoutSeconds = 600
```
Comment (thebigjc) [inline main.go]: There are a lot of magic durations in here. They should probably have Env overrides.

**Comment by thebigjc** [comment]
:shipit:

**Comment by jharlap** [comment]
Squashed.

**Pull request merged**